README.md

```diff
@@ -14,3 +14,40 @@ continuously on a private registry at r.j3ss.co for public download. (You're
 welcome.)
 
 You may also want to checkout my [dotfiles](https://github.com/jfrazelle/dotfiles), specifically the aliases for all these files which are here: [github.com/jfrazelle/dotfiles/blob/master/.dockerfunc](https://github.com/jfrazelle/dotfiles/blob/master/.dockerfunc).
+
+### Running on macOS
+
+> Note added by [Alex Ellis](https://github.com/alexellis)
+
+It is possible to run graphical apps for Linux through Docker on your Mac - here's a few advantages:
+
+* To access to newer versions of software
+* To test various versions of the same software simultaneously
+* To use tools which may not be ported to macOS yet
+
+For sandboxing an application:
+* To tighten up on security
+* or to isolate and/or spy on network traffic
+
+Follow this guide for more information:
+
+* [Bring Linux apps to the Mac Desktop with Docker](http://blog.alexellis.io/linux-desktop-on-mac/)
```

Comment:
Maybe we should make a list of the posts on it, @chanezon wrote a great one too https://github.com/chanezon/docker-tips/tree/master/x11